src/unsupported/pbs_rescquery.3B

```diff
@@ -1,5 +1,5 @@
 .\"
-.\" Copyright (c) 1999-2016 Altair Grid Technologies, LLC.  All rights reserved. 
+.\" Copyright (c) 1999-2017 Altair Grid Technologies, LLC.  All rights reserved. 
```

Comment:
The copyright notice in this file does not conform to the standard. This is not due to anything you have changed. Please remove the notice (including the paragraph below) and replace it with the notice contained in doc/man1/qsub.1B